src/pretix/control/forms/vouchers.py

```diff
@@ -179,9 +179,7 @@ def clean(self):
         return data
 
     def save(self, commit=True):
-        super().save(commit)
-
-        return ['item']
```

Comment:
I'm not sure why this was there. Removing it does not break any tests, but I'm still a little concerned.

Reply:
This looks weird, I'm not sure why it was there either.